config/routes.rb

```diff
@@ -214,6 +214,7 @@
   get 'u/:username/'                       => 'users#show',       as: :user
 
   get 'dashboard'                          => 'application#boot_angular_ui', as: :dashboard
+  get 'dashboard/show_muted'               => 'application#boot_angular_ui', as: :muted
```

Comment:
As defines a route helper method. (`muted_path`), which we don't use anywhere. So no as.